app/views/components/_important-metadata.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+  items = local_assigns[:items] || {}
+  items = items.reject { |k,v| v.nil? }
+  items = items.merge(items) { |k,v| Array(v).join(",").html_safe }
+-%>
+<% if items.any? -%>
+  <div class="app-c-important-metadata">
+    <% items.each do |title, definition| -%>
+      <p><%= title %>: <strong><%= definition %></strong></p>
```

Comment:
Might be good as a list see original metadata component

Reply:
Updated here: https://github.com/alphagov/government-frontend/pull/541/commits/0d340f2d1951428c0a8faabaddf3ca31538258d5